helix-core/src/main/java/org/apache/helix/manager/zk/zookeeper/ZkClient.java

```diff
@@ -1166,6 +1432,12 @@ private void acquireEventLock() {
     }
     try {
       while (true) {
+        // Because ConnectionLossException and SessionExpiredException are caught but not thrown,
```

Comment:
nit, just my personal opinion, this is too verbose. This var is very clear to be recording and logging what exception we see during the operation.

Reply:
We don't expose the exception in the message. Since we just need the code in the message, I will change it to a code type so we don't record the exception itself.

Reply:
@jiajunwang, let us document this invariant so that people who later change the code knows what is the assumption here.

A lot of times later people who wrote the code would forget about the invariant they assumed when they wrote the code.